### PR TITLE
Create cc wallet command

### PR DESCRIPTION
**Pull request by jespino**

### CODE REVIEW

**Comment by jespino** [comment]
I added also the rate limited wallet here :)

**Comment by justinengland** [comment]
can you please resubmit this PR to either main, or if it needs to be merged before 3/24/21, new-main, this will allow us to move forward with the new git branching strategy

**Comment by jespino** [comment]
Sure, I'll do.

File: chia/cmds/wallet_funcs.py [hunk 1]
```diff
@@ -230,3 +230,19 @@ async def execute_with_wallet(wallet_rpc_port: int, fingerprint: int, extra_para
...
     wallet_client.close()
     await wallet_client.await_closed()
+
+
+async def create_new_wallet(args: dict, wallet_client: WalletRpcClient, fingerprint: int) -> None:
+    data = await wallet_client.create_new_wallet(args["wallet_type"], args["data"])
+    if data and data["success"]:
+        if args["wallet_type"] == "cc_wallet" and args["data"]["mode"] == "new":
+            print("New colour created: ", data["colour"])
+        elif args["wallet_type"] == "cc_wallet" and args["data"]["mode"] == "existing":
```
Comment (matt-o-how) [inline chia/cmds/wallet_funcs.py:240]: This if is repeated, structuring can factor it out

**Review by myles1993 — DISMISSED** [review]

File: chia/cmds/wallet_funcs.py (hunk 1 above)
Comment (atyhieu) [reply chia/cmds/wallet_funcs.py:240]: chia/cmds/wallet_funcs.py

**Review by atyhieu — APPROVED** [review]

**Review by atyhieu — DISMISSED** [review]

File: chia/cmds/wallet_funcs.py (hunk 1 above)
Comment (jespino) [reply chia/cmds/wallet_funcs.py]: Done

**Comment by hoffmang9** [comment]
Bump

**Comment by jespino** [comment]
@hoffmang9 this one is ready again, merged the main branch and some small tweaks around cli parameters to make them more consistent with the rest of the wallet cli commands.

**Comment by mariano54** [comment]
This is something that we might want to come back to, since CATs will be coming soon

**Comment by mariano54** [comment]
Closing due to being done in main now.

**Pull request closed without merging**